zephyr/wrapper.c

```diff
@@ -72,18 +72,32 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
 
-#define HEAPMEM_SHARED_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + \
-				HEAP_RUNTIME_SHARED_SIZE + HEAP_SYSTEM_SHARED_SIZE)
+#if (CONFIG_HP_MEMORY_BANKS <= 8)
+/* APL */
+#define	HEAP_SIZE	0x30000
+#elif (CONFIG_HP_MEMORY_BANKS == 16)
+/* JSL */
+#define	HEAP_SIZE	0x80000
+#elif (CONFIG_HP_MEMORY_BANKS == 30)
+/* TGL-H */
+#define	HEAP_SIZE	0x100000
+#elif (CONFIG_HP_MEMORY_BANKS >= 46)
+/* CNL/ICL/TGL */
+#define	HEAP_SIZE	0x200000
+#endif
```

Comment:
Hmm, this is a shortterm workaround (e.g.  Zephyr's libc already has similar code to figure out the heap size:
zephyr/lib/libc/newlib/libc-hooks.c ), so I'd prefer a very simple definition and add a TODO comment to clearly mark this is a stopgap.